jsonschema-maven-plugin/src/main/java/com/github/victools/jsonschema/plugin/maven/SchemaGeneratorMojo.java

```diff
@@ -56,38 +60,41 @@
         requiresDependencyCollection = ResolutionScope.COMPILE)
 public class SchemaGeneratorMojo extends AbstractMojo {
 
-    @Parameter(defaultValue = "${project}", required = true, readonly = true)
-    MavenProject project;
+    /**
+     * Full name of the classes for which the JSON schema will be generated.
+     */
+    @Parameter(property = "classNames")
+    private String[] classNames;
 
     /**
-     * Full name of the class for which the JSON schema will be generated.
+     * Full name of a package for which for all classes in the package JSON schemas will be generated.
      */
-    @Parameter(property = "className", required = true)
-    private String className;
+    @Parameter(property = "packageNames")
+    private String[] packageNames;
 
     /**
      * The directory path where the schema files are generated.
      * <br>
      * By default, this is: {@code src/main/resources}
      */
-    @Parameter(property = "schemaFilePath")
+    @Parameter
```

Comment:
I don't see a reason to prevent the `schemaFilePath` to be used via the command-line. I had only removed it from the modules and options because those objects would be very hard to use on the command-line anyway.
```suggestion
    @Parameter(property = "schemaFilePath")
```

Reply:
Agree. My bad.
Fixed.